packages/styling/src/interfaces/IFontStyles.ts

```diff
@@ -4,16 +4,16 @@ import { IRawStyle } from '../interfaces/index';
  * UI Fabric font set.
  */
 export interface IFontStyles {
```

Comment:
Turns out I don't need this change to turn on strict null checks. Though it is worth following up if we can be more strict on this type.